instagram/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import PostContent from "../PostContent/PostContent";
+import "./PostContainer.css";
+import PropTypes from 'prop-types';
+
+const PostContainer = props => {
+  return (
+    <div className="post-container">
+      {props.data.map(item => {
+        return <PostContent key={item.username} data={item} />;
+      })}
+    </div>
+  );
+};
+
+PostContainer.propTypes = {
+    data: PropTypes.array.isRequired
```

Comment:
Here, instead of just `PropTypes.array`, you could check for a the contents of the array, to make sure each item is an object and has a username property (and whatever else it's supposed to have).